### PR TITLE
Clean up e2e vpa tests

### PR DESCRIPTION
**Pull request by jbartosik**

### CODE REVIEW

File: vertical-pod-autoscaler/e2e/v1beta2/admission_controller.go [hunk 1]
```diff
@@ -116,7 +116,7 @@ var _ = AdmissionControllerE2eDescribe("Admission-controller", func() {
 		}
 	})
 
-	ginkgo.It("caps request according to container max limit set in LimitRange", func() {
+	ginkgo.It("caps request according to container max limit set in container LimitRange", func() {
```
Comment (bskiba) [inline vertical-pod-autoscaler/e2e/v1beta2/admission_controller.go]: Why do we need container twice?

File: vertical-pod-autoscaler/e2e/v1beta2/common.go [hunk 2]
```diff
@@ -234,18 +234,18 @@ func NewVPA(f *framework.Framework, name string, targetRef *autoscaling.CrossVer
...
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	gomega.Expect(err).NotTo(gomega.HaveOccurred(), "unexpected error creating VPA")
 }
 
 // ParseQuantityOrDie parses quantity from string and dies with an error if
 // unparsable.
 func ParseQuantityOrDie(text string) resource.Quantity {
 	quantity, err := resource.ParseQuantity(text)
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	gomega.Expect(err).NotTo(gomega.HaveOccurred(), "unexpected error parsing qunantity: %s", text)
```
Comment (bskiba) [inline vertical-pod-autoscaler/e2e/v1beta2/common.go]: s/qunatity/quantity/

File: vertical-pod-autoscaler/e2e/v1beta2/updater.go [hunk 3]
```diff
@@ -158,7 +158,7 @@ var _ = UpdaterE2eDescribe("Updater", func() {
 		CheckNoPodsEvicted(f, MakePodSet(podList))
 	})
 
-	ginkgo.It("observes pod max in LimitRange", func() {
+	ginkgo.It("observes pod max in pod LimitRange", func() {
```
Comment (bskiba) [inline vertical-pod-autoscaler/e2e/v1beta2/updater.go]: Same, why pod twice?

File: vertical-pod-autoscaler/e2e/v1beta2/common.go (hunk 2 above)
Comment (jbartosik) [reply vertical-pod-autoscaler/e2e/v1beta2/common.go]: Done.

File: vertical-pod-autoscaler/e2e/v1beta2/admission_controller.go (hunk 1 above)
Comment (jbartosik) [reply vertical-pod-autoscaler/e2e/v1beta2/admission_controller.go]: We don't

File: vertical-pod-autoscaler/e2e/v1beta2/updater.go (hunk 3 above)
Comment (jbartosik) [reply vertical-pod-autoscaler/e2e/v1beta2/updater.go]: I missed this when cleaning up the PR

**Comment by jbartosik** [comment]
/assign @bskiba 
I checked that " observes pod min in LimitRange" test started passing when I modified VPA to set lower bound to min(target, lower bound) and similarly for upper bound.

**Pull request merged**